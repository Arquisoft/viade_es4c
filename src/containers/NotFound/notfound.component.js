import React from 'react';
import {Link, NavLink} from 'react-router-dom';

const NotFoundComponent = () => {
	return (
	    <div className="ErrorImage">
			<NavLink to="/">
				<img src="/img/error404.jpg" alt="error" height="50%" width="50%"/>
			</NavLink>
			<p style={{textAlign: "center"}}>
				<Link to="/">Go back Home </Link>
			</p>
		</div>
	);
};

export default NotFoundComponent;
