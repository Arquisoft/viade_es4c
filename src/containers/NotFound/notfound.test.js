import React from 'react';
import { shallow } from 'enzyme';
import NotFound from "./notfound.component";

describe('NotFound', () => {
  it('should render correctly in "debug" mode', () => {
    const component = shallow(<NotFound debug />);
  
    expect(component).toMatchSnapshot();
  });
});
