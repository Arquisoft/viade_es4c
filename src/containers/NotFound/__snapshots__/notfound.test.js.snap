// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFound should render correctly in "debug" mode 1`] = `
<div
  className="ErrorImage"
>
  <NavLink
    to="/"
  >
    <img
      alt="error"
      height="50%"
      src="/img/error404.jpg"
      width="50%"
    />
  </NavLink>
  <p
    style={
      Object {
        "textAlign": "center",
      }
    }
  >
    <Link
      to="/"
    >
      Go back Home 
    </Link>
  </p>
</div>
`;
